### PR TITLE
Longest consecutive sequence in Java

### LINKED ISSUE #6191: Longest Consecutive Sequence
## 💥 Proposal

I would like to add the code for Longest Consecutive Sequence in Java under the list of CP questions because this is a good question and is likely to be faced during interview

Ques - Return the length of longest consecutive elements sequence in an unsorted array

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

yes

### PR DESCRIPTION
**Pull request by apoorva0107**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Return the length of the longest consecutive elements sequence in an unsorted array.
I have added the Java code of this problem under the section of cp in Java. I have also added the time and space complexities along with sample test cases. 

### Checklist

- [ :heavy_check_mark: ] I've read the contribution guidelines.
- [ :heavy_check_mark: ] I've checked the issue list before deciding what to submit.
- [ :heavy_check_mark: ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

fixes #6191

### CODE REVIEW

**Comment by apoorva0107** [comment]
Fixes #6191

**Comment by apoorva0107** [comment]
fixes #6191

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Add the `.java` extension to the file.

**Comment by apoorva0107** [comment]
@HarshCasper I made the changes, please review.

**Review by ankitaggarwal23 — COMMENTED** [review]
Read the instructions mentioned on Gssoc discord yesterday. Do the changes as per that.

**Review by ankitaggarwal23 — COMMENTED** [review]
It is not a cp problem kindly add it to appropriate section

**Comment by apoorva0107** [comment]
@ankitaggarwal23 Which section should I add it to? Should I put this in 'ds' section or 'other'? Simple questions like "removing an element from array" and "reverse string" are also included in the 'cp' section so I thought this belongs there.

**Comment by ankitaggarwal23** [comment]
> @ankitaggarwal23 Which section should I add it to? Should I put this in 'ds' section or 'other'? Simple questions like "removing an element from array" and "reverse string" are also included in the 'cp' section so I thought this belongs there.

No we are adding files in their proper section.  So add your file in proper section

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Shift your code file also to other section. still in cp section

**Comment by apoorva0107** [comment]
@ankitaggarwal23 trying to figure that out. Please help.

**Comment by apoorva0107** [comment]
@ankitaggarwal23 when I pull this into my local device it shows everything up to date, but why does it still have conflict?

File: Java/other/Longest_Consecutive_Squence.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
+/*
+Problem Description: For a given unsorted array, your task is to return the length of longest sub-sequence such that elements in the subsequence are consecutive integers.
+For example: For an array [3, 6, 10, 2, 4] the longest sequence of consecutive elements is 2, 3, 4 so the length will be 3.
+*/
```
Comment (ankitaggarwal23) [inline Java/other/Longest_Consecutive_Squence.java]: Rewrite it like this
```
/*
Problem Description: For a given unsorted array, your task is to return
the length of longest sub-sequence such that elements in the subsequence
are consecutive integers.

For example: For an array [3, 6, 10, 2, 4] the longest sequence of 
consecutive elements is 2, 3, 4 so the length will be 3.
*/
```

File: Java/other/Longest_Consecutive_Squence.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+Sample Case 2
+Input-
+Enter number of elements:
+11
+Enter elements:
+9 1 4 7 3 -1 0 5 8 -1 6
+
+Output-
+Length of longest consecutive sequence: 
+7
```
Comment (ankitaggarwal23) [inline Java/other/Longest_Consecutive_Squence.java]: Remove extra test case

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

File: Java/other/Longest_Consecutive_Squence.java (hunk 2 above)
Comment (apoorva0107) [reply Java/other/Longest_Consecutive_Squence.java]: @ankitaggarwal23 made the changes

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**